src/utils/jwt.js

```diff
@@ -0,0 +1,30 @@
+const jwt = require('jsonwebtoken');
+
+const dotenv = require('dotenv');
+
+dotenv.config();
+const secretKey = process.env.JWT_KEY;
+/**
+ *
+ */
+export default class jwtHelper {
+  /**
+   * @param {object} payload - The details to be signed
+   * @param {string} secret - The JWT secret key
+   * @returns {string} The JWT signed token
+   */
+  static async generateToken(payload, secret = secretKey) {
```

Comment:
@Osasbaby are you using all the payload details to generate token or what??

Reply:
Yes
The details passed into the function will be used to generate token
If you pass in email username  firstname and last name that is what will be used to generate the token

Reply:
@Osasbaby you can't use all the details in the payload to generate a token, use id

Reply:
done